frontend/src/components/editProject/EditCollaboratingOrganizations.js

```diff
@@ -0,0 +1,53 @@
+import { List } from "@material-ui/core";
+import React from "react";
+
+import MiniOrganizationPreview from "../organization/MiniOrganizationPreview";
+import AutoCompleteSearchBar from "../search/AutoCompleteSearchBar";
+
+export default function EditCollaboratingOrganizations({
+  baseUrl,
+  handleAddCollaboratingOrg,
+  handleDeleteCollaboratingOrg,
+  organizationPreviewClass,
+  organizationPreviewsContainerClass,
+  searchBarClass,
+  selectedCollaboratingOrgs,
+  texts,
+}) {
+  const renderSearchOption = (option) => {
+    return <>{option.name}</>;
+  };
+
+  return (
+    <>
+      {/* TODO: Currently allows any org to be added, maybe needs some verification on selected orgs part*/}
```

Comment:
I think this is very important feature to have. I don't think we should allow any organization to be added. Lets make sure that feature requirements are clarified before we move forward merging this PR.

Reply:
We should maybe look to create an issue for this as its own PR, with also making this check for project creation?